yang-semver/draft-ietf-netmod-yang-semver.txt

```diff
@@ -374,7 +374,39 @@ Internet-Draft                 YANG Semver                  October 2022
    modifier strings.  E.g., artifact version "1.2.3_non_compatible" MUST
    NOT be defined if artifact version "1.2.3" has already been defined.
 
-3.3.1.  YANG Semver with submodules
+3.3.1.  Branching Limitations with YANG Semver
```

Comment:
I wonder if this perhaps belongs more in a guidelines appendix, where we could also add more illustrative examples of allowed branching and disallowed branching ?